react/components/Arrow.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import styles from './styles.css'
+import { IconCaret } from 'vtex.dreamstore-icons'
+
+const Arrow = props => {
+  const {
+    tag: RootTag,
+    orientation,
+    size,
+    classes,
+    ...restProps
+  } = props
+
+  const containerStyle = size ? { height: size + 10, width: size + 10 } : {}
+
+  return (
+    <RootTag
+      className={classnames(styles.arrowContainer, classes.container, 'bg-transparent bn absolute pa2')}
+      style={containerStyle}
+      {...restProps}
+    >
+      <IconCaret className={classes.arrow} size={size} orientation={orientation} />
```

Comment:
🤔Could you not receive this Icon pre-configured via props?  This way, you may not need `classes`, `size`, and `orientation` props.